README.md

```diff
@@ -3,59 +3,64 @@
 
 ## Installation
-- latest_version = 0.0.1
+- latest_version = 0.0.37
 - minSdkVersion = 28
 
 ### Gradle
 
-To integrate VeraSDK into your android project using gradle, build.gradle of app module:
+To integrate VeraSDK into your android project using gradle: 
+on your root build.gradle:
 
 ```groovy
- implementation 'com.github.resonai:vera-android-sdk:0.0.1'
- 
 repositories {
-    maven {
-        url "https://oss.sonatype.org/content/repositories/snapshots"
-    }
     maven { url 'https://jitpack.io' }
 }
-
-
-packagingOptions {
-    pickFirst "**/libfbjni.so"
-    pickFirst 'lib/x86/libc++_shared.so'
-    pickFirst 'lib/x86_64/libc++_shared.so'
-    pickFirst 'lib/armeabi-v7a/libc++_shared.so'
-    pickFirst 'lib/arm64-v8a/libc++_shared.so'
-}
 ```
 
-Then,sync project:
+Add the dependency to your module
+```groovy
+    implementation 'com.github.resonai:vera-android-sdk:0.0.37'
+```
 
+Then, sync project:
 
 ## Usage 
-  Create a configuration object and start sdk  for both case 
+  Create a configuration object and start sdk for both case
 
-
-With Login :
-    To track user information you can  pass user data.
+```groovy
+VeraConfiguration.Builder veraBuilder = new VeraConfiguration.Builder
+        // (this) Activity used
+        (getSupportFragmentManager(), R.id.container) // pass container for fragment
+        .setClientAppID("vera_client_app")) // Your custom Vera platform ID.
+        .setSiteIDs("siteId1", "siteId2");
+        .setShowCloseButton(shouldShow) // A boolean value of whether the SDK should show it's own close button - default to true
+        .setHideHeader(false) // A boolean value for the SDK to hide or show the default header. Defaults to `false`
+        .setDeeplinkPrefix(veraDomain) //A string value defining your custom "deepLinkPrefix". Used when generating deep links that you can pass into Vera and should open your app.
+        .setLanguage(sdkLanguage) // Choose vera language
+        .onRequestRefreshToken(this::fetchUserToken) // Listener triggered when user token need refresh
+        .onMessageListener(this::onMessage) // Listener to messages from vera and implement logic for them
+        .onCloseListener(this::closeVeraSdk) // Listener triggered when vera closed
+```
+You can Start vera with user data :
 
 ```groovy
 
-        VeraConfiguration.Builder()
-               .setClientAppID("vera_client_app")
-               .setLanguage(Languages.EN)
-               .startWithLogin(this, "userName", "userId", "token");
+     builder.startWithLogin(useName(), userId, token);
 ```
-Without login as :
+
+You can also start vera without user data :
 
 ```groovy
-     VeraConfiguration.Builder()
-                  .setClientAppID("vera_client_app")
-                  .setLanguage(Languages.EN)
-                  .startWithoutLogin(this);
+
+     builder.startWithoutLogin();
 ```
 
+```groovy
+
+    when you need to send vera data after vera already exist you can use setDeeplinkComponent:
```

Comment:
* make "Vera" capitalized (other comments in this file too if there are any)
* capitalize "When"
* "after Vera already exist**s**, you can" etc. - is it clear what you mean by "exists" as opposed to say "is running" ?

Reply:
* backtick `` `setDeelinkComponent` ``